schematic/version.py

```diff
@@ -1,4 +1,4 @@
 """Sets the version of the package"""
 import importlib.metadata
 
-__version__ = importlib.metadata.version("schematic")
+__version__ = importlib.metadata.version("schematicpy")
```

Comment:
Going forward will we still want to manually update the version in the `pyproject.toml`?